probe/kubernetes/client.go

```diff
@@ -1,12 +1,14 @@
 package kubernetes
 
 import (
+	"fmt"
 	"io"
 	"strconv"
 	"sync"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/weaveworks/common/backoff"
```

Comment:
Please move this to a separate block above.

As a convention we create 3 import blocks:

1. Standard library imports
2. Weaveworks imports
3. External imports

See for instance https://github.com/weaveworks/scope/blob/master/probe/overlay/weave.go